tests/Unit/RollbackServiceTest.php

```diff
@@ -124,26 +124,15 @@ public function testRollbackOlderThan(): void {
 		$userFolder5 = $this->createMock(Folder::class);
 		$userFolder6 = $this->createMock(Folder::class);
 
-		$this->rootFolder->expects($this->at(0))
-			->method('getUserFolder')
-			->with('user2')
-			->willThrowException(new \Exception('User not found'));
-		$this->rootFolder->expects($this->at(1))
-			->method('getUserFolder')
-			->with('user3')
-			->willReturn($userFolder3);
-		$this->rootFolder->expects($this->at(2))
-			->method('getUserFolder')
-			->with('user4')
-			->willReturn($userFolder4);
-		$this->rootFolder->expects($this->at(3))
-			->method('getUserFolder')
-			->with('user5')
-			->willReturn($userFolder5);
-		$this->rootFolder->expects($this->at(4))
-			->method('getUserFolder')
-			->with('user6')
-			->willReturn($userFolder6);
+		$this->rootFolder->method('getUserFolder')
```

Comment:
No expects() ? Does that do anything? I would expect at least `expects($this->any())`

Reply:
No `exepcts` is equivalent to `expects($this->any())`. I could add $this->atLeastOnce() but that pretty useless and map too much the tests to the implementation